core/analysis/collation_token_stream.cpp

```diff
@@ -112,8 +226,21 @@ bool make_vpack_config(
   VPackObjectBuilder object(builder);
   {
     // locale
-    const auto& locale_name = locale_utils::name(options.locale);
-    builder->add(LOCALE_PARAM_NAME, VPackValue(locale_name));
+    {
```

Comment:
As general suggestion:  Let's move all locale object serializing / deserializing to the separate compilation unit. So we could use it later for other analyzers (like text, norm and so on)